src/components/playback/mediasession.js

```diff
@@ -49,121 +30,96 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
+
             return connectionManager.getApiClient(item.ServerId).getScaledImageUrl(item.AlbumId, options);
         }
 
         return null;
     }
 
-    function pushImageUrl(item, imageOptions, list) {
-        var url = seriesImageUrl(item, imageOptions) || imageUrl(item, imageOptions);
+    function pushImageUrl(item, imageOptions = {}) {
+        const url = seriesImageUrl(item, imageOptions) || imageUrl(item, imageOptions);
 
         if (url) {
-            var height = imageOptions.height || imageOptions.maxHeight;
+            const height = imageOptions.height || imageOptions.maxHeight;
 
-            list.push({
+            return {
                 src: url,
                 sizes: height + 'x' + height
-            });
+            };
         }
     }
 
-    function getImageUrls(item) {
+    function getImageUrls(item, imageSizes = [96, 128, 192, 256, 384, 512]) {
+        const list = [];
 
-        var list = [];
-
-        pushImageUrl(item, {height: 96}, list);
-        pushImageUrl(item, {height: 128}, list);
-        pushImageUrl(item, {height: 192}, list);
-        pushImageUrl(item, {height: 256}, list);
-        pushImageUrl(item, {height: 384}, list);
-        pushImageUrl(item, {height: 512}, list);
+        imageSizes.forEach((size) => {
+            list.push(pushImageUrl(item, {height: size}));
+        });
 
         return list;
     }
 
     function updatePlayerState(player, state, eventName) {
         // Don't go crazy reporting position changes
-        if (eventName == 'timeupdate') {
+        if (eventName === 'timeupdate') {
             // Only report if this item hasn't been reported yet, or if there's an actual playback change.
             // Don't report on simple time updates
             return;
         }
 
-        var item = state.NowPlayingItem;
+        const item = state.NowPlayingItem;
 
         if (!item) {
             hideMediaControls();
             return;
         }
 
-        if (eventName == 'init') { // transform "init" event into "timeupdate" to restraint update rate
+        if (eventName === 'init') { // transform "init" event into "timeupdate" to restraint update rate
             eventName = 'timeupdate';
         }
 
-        var isVideo = item.MediaType === 'Video';
-        var isLocalPlayer = player.isLocalPlayer || false;
+        const isVideo = item.MediaType === 'Video';
+        const isLocalPlayer = player.isLocalPlayer || false;
 
         // Local players do their own notifications
         if (isLocalPlayer && isVideo) {
             return;
         }
 
-        var playState = state.PlayState || {};
-        var parts = nowPlayingHelper.getNowPlayingNames(item);
-        var artist = parts[parts.length - 1].text;
-        var title = parts.length === 1 ? '' : parts[0].text;
-        var albumArtist;
-
-        if (item.AlbumArtists && item.AlbumArtists[0]) {
-            albumArtist = item.AlbumArtists[0].Name;
-        }
```

Comment:
so this is now lost?

Reply:
It never worked, since it's only used in the MediaSessions object that is passed to the browser and, afaik, that has never supported album artists or some of the other properties set before...